### DISCUSSION TITLE
New Stat Panel scalability improvement.

### DISCUSSION THREAD

**Discussion by scroodj** [grafana/grafana#38363] [Dashboards, Panels & Visualizations]
Hello,

First of all i would like to give my gratitude to you guys for that you are doing.
Thank you for a new Stat Panel arrived with 6.6 release. I see a big potential of using this panel.
Nevertheless i want to suggest some improvements.

When it comes to using it with just a few numbers of series it looks good:
![stat_panel_6_series](https://user-images.githubusercontent.com/6392717/74521332-8e654680-4f19-11ea-9bee-a575707b1212.PNG)

but with increasing number of output values(multiple series) it becomes not user friendly:
![stat_panel_multiple_series](https://user-images.githubusercontent.com/6392717/74521521-cec4c480-4f19-11ea-8c77-24b3c207c159.PNG)
 

**What would you like to be added**:
1. Use Vertical Scalability (multiply rows) instead of using Horizontal Scalability within a same panel. Example:
![stat_panel_multiple_series_multiple_rows](https://user-images.githubusercontent.com/6392717/74521704-2400d600-4f1a-11ea-9888-205ecbd7c963.PNG)
2. Control over the font size.
3. Ability to fix individual cube size.

**Comment by torkelo** [comment]
We plan to add a grid layout mode, but not manual font size , that will not help

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
